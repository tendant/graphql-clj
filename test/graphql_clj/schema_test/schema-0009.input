enum Episode {
    NEWHOPE
    EMPIRE
    JEDI
 }