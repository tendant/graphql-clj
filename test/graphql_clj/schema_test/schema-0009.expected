{:enums {:Episode {:values ["NEWHOPE" "EMPIRE" "JEDI"]}}}
